common/common-profile.c

```diff
@@ -0,0 +1,78 @@
+/*
+
+Copyright (c) 2025, MithicSpirit
```

Comment:
We plan on changing the copyright notice to state "the GameMode contributors" to recognise the contributions from the community.

It will get a little messy if each source file has a specific individual contributors name. In particular, this won't match the notice in the LICENSE.txt file, and reproductions of the copyright notice on binary distrubutions would be similarly complicated.

This won't affect your existing copyright on this code. Additionally, the license of the project and other code will remain the same with full attribution still available on the project page.

With all that said, could you change this line to "Copyright (c) 2025, the GameMode contributors" before I merge this pull request?